_kasu_smoke.py

```
@@ -0,0 +1,14 @@
+"""Smoke test for the Kasu PR reviewer. DO NOT MERGE - safe to delete."""
+
+
+def average_price(items):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `average_price`

(ANN202)